department-of-veterans-affairs/abd-vro
# Rename service-data-access to svc-lighthouse-api folder

## What was the problem?

`service-data-access` folder name is too general.

## How does this fix it?[^1]

Rename it to `svc-lighthouse-api`. The only changes are:
- settings.gradle
- build.gradle
- renaming the directory

This is only renaming the folder for now -- we'll rename the Docker image in a separate PR.

## How to test this PR
All images are still created

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
